2-virtual-machines-containers/2.1-golden-images.md

```diff
@@ -5,14 +5,35 @@
 ![](img2/linux_light.svg ':size=100x100 :class=light-mode-icon')
 ![](img2/linux_dark.svg ':size=100x100 :class=dark-mode-icon')
 
+?> If you have an m1 mac checkout virtualbox beta build for arm64. ( Note that as of 23/02/2023 the virtualbox beta build for arm64 did not work)
+
+?> assuming you have an m1 mac & the virtualbox beta is still not operational, the only m1 compatible section is the compatibility section.
+
+### M1 Compatibility Excersize
+
+1. You will need to install the utm project. assuming you have brew  `brew install --cask utm` should work
+
+2. You will also need an ARM64 linux iso; ideally we want to use the centos9 iso located [here](https://mirrors.centos.org/mirrorlist?path=/9-stream/BaseOS/aarch64/iso/CentOS-Stream-9-latest-aarch64-dvd1.iso&redirect=1&protocol=https) make sure to select the ARM64 version.
+
+3. Once you have the iso downloaded open the UTM gui.
+    1. create a new vm, select virtualize
+    2. when prompted for "virtualization engine", you need to give your vm the iso you downloaded. probably in the local Downloads directory.
+    ![](img2/UTM_iso_selection.png)
+    3. Once you have an iso selected, continue to the end (default vm hard settings will work fine).
+    4. Launch your vm, go through the centos9 install. (defaults/automatic should be fine for everything)
+    5. Once you asks you to reboot shut the vm down and unmount your iso before rebooting.
+    ![](img2/UTM_iso_unmount.png)
+
+4. Congradulations you have a centos9 image, you can create ssh keys manually with the same steps as below
+
 #### Exercise 1: Manual Build
 
 For this exercise, you will create a Golden Image by manually installing and configuring CentOS 7 in a VirtualBox VM.
 
-1. Download the [CentOS 7](https://www.centos.org/download/) ISO for a 64-bit x86 machine.
- 
+1. Download the [CentOS 9](https://mirrors.centos.org/mirrorlist?path=/9-stream/BaseOS/aarch64/iso/CentOS-Stream-9-latest-aarch64-dvd1.iso&redirect=1&protocol=https) ISO for a 64-bit x86 machine.
```

Comment:
Same concern about a specific link to a mirror download.

Reply:
fixed here: https://github.com/liatrio/devops-bootcamp/pull/232/commits/4216c2744c7c19579530aaeddbf63b1d30d7636c